src/langs/fr.json

```diff
@@ -511,5 +511,9 @@
   "DELETE_GUESTS_MODAL_CONTENT": "Je confirme la suppression <1>définitive</1> des lecteurs suivants et de leurs données :",
   "DELETE_GUESTS_MODAL_DELETE_BUTTON": "Supprimer définitivement",
   "ADD_TAG_OPTION_BUTTON_TEXT": "Ajouter {{value}}",
-  "CREATE_FOLDER_BUTTON_TEXT": "Nouveau Dossier"
+  "CREATE_FOLDER_BUTTON_TEXT": "Nouveau Dossier",
+  "TAGS_DESCRITPION": "Les tags sont des bouts d'informations qui peuvent être attachés aux éléments. Ils permettent de décrire le sujet, contexte, projet, ou l'audience attendue pour leurs éléments. Ils aident les autres utilisateurs à trouver votre contenu quand ils cherchent dans la librarie. Vous pouvez en apprendre plus sur les tags dans <1>la documentation</1>.",
+  "TAGS_DISCIPLINE_PLACEHOLDER": "Exemple: mathematics, history, biology",
+  "TAGS_LEVEL_PLACEHOLDER": "Exemple: elementary, high school, undergraduate",
```

Comment:
```suggestion
  "TAGS_LEVEL_PLACEHOLDER": "Example: école primaire, université, cycle préparatoire",
```

Reply:
In french it's "exemple" ;) But indeed I forgot to translate the tags.

Reply:
Ah oops haha